.kitchen.cloud.yml

```diff
@@ -17,14 +17,15 @@ provisioner:
   name: chef_solo
 
 platforms:
-  - name: centos-6.6
-    driver_plugin: openstack
-    driver_config:
-      username: centos
-      image_ref: "CentOS 6.6"
```

Comment:
Any particular reason to remove 6.6 support? We still have quite a bit of 6.6 deployed.